django/db/backends/oracle/base.py

```diff
@@ -200,10 +200,15 @@ def _connect_string(self):
                              settings_dict['PASSWORD'], dsn)
 
     def get_connection_params(self):
+        kwargs = {
+            'encoding': 'UTF-8',
+            'nencoding': 'UTF-8',
+        }
         conn_params = self.settings_dict['OPTIONS'].copy()
         if 'use_returning_into' in conn_params:
             del conn_params['use_returning_into']
-        return conn_params
+        kwargs.update(conn_params)
```

Comment:
Makes sense. I'd say that a test could be useful but since this isn't going to master, I don't mind if it's omitted.